config/routes.rb

```diff
@@ -56,7 +57,7 @@
 
   post "consume_paddle_webhook", to: "admin/billing#consume_paddle_webhook"
 
-  resources :boards, path: "" do
+  resources :boards, path: "/" do
```

Comment:
```suggestion
  resources :boards, path: "" do
```
Can we remove this change?

Reply:
@pranav7 not making any change in this as you are working on this.